docs/articles/documentation/extending-testcafe/custom-reporter/managing-reporter-project.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+title: Managing Reporter Project
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/managing-reporter-project/
+---
+# Managing Reporter Project
```

Comment:
I'm not quite sure that testing, preview and publishing is actually *managing*.
Isn't it a part of the *creation* process?
Probably not, but anyway, to my mind, managing is something that becomes after the reporter is ready and published.